README.md

```diff
@@ -0,0 +1,233 @@
+<a name="readme-top"></a>
+<!--
+<div align="center">
+  You are encouraged to replace this logo with your own! Otherwise you can also remove it. 
+  <h3><b>Microverse README Template</b></h3>
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+</div> 
+-->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [OOP-school-library] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
```

Comment:
- [ ] Please delete the template placeholder text: `Describe your project in 1 or 2 sentences.` in your readme, this will make your readme more professional 👍🏿